src/engines/kotor/creature.cpp

```diff
@@ -317,7 +307,10 @@ void Creature::createPC(CharacterGenerationInfo *info) {
 	}
 
 	// set the specific class to level 1
-	_levels[info->getClass()] = 1;
+	_levels.clear();
+	_levels.emplace_back();
```

Comment:
std::vector::emplace_back() is a C++11 feature, don't use that yet.

Just resize(1) instead of clear+emplace. It's shorter, too :P